src/Lumi/Components2/Slat.purs

```diff
@@ -28,10 +29,15 @@ type SlatProps
     , interaction :: Maybe SlatInteraction
     )
 
+data SlatInteractionType
+  = BorderInteraction
+  | BackgroundInteraction
+
 type SlatInteraction
   = { onClick :: Effect Unit
     , tabIndex :: Int
     , href :: Maybe URL
+    , _type :: Maybe SlatInteractionType
```

Comment:
Not sure if adding this prop is the best way to implement this. We're just moving the problem of extending a record being a breaking change out of the `Slat` props and into `SlatInteraction`.

I believe it'd make more sense to, instead define a modifier `_interactiveBackground` that does just what `_interactive` already does plus adding the `_interactiveBg` modifier.

Reply:
Something like this? https://github.com/lumihq/purescript-lumi-components/pull/130/commits/61a201b457ae0cef467307e27456590607230f07

@hdgarrood unfortunately we need to support both, as design has asked for the option of both a border color on hover vs. a background color on hover. I believe when the `Slat` is sitting independently a border color hover interaction is desired, whereas when the `Slat` is a "list/table-like" (i.e. non-spaced out) structure background-color hover interaction is preferred.

Reply:
Makes sense, thanks.

Reply:
@kimispencer Yes, that looks good!